### PR TITLE
Editor: Update WordPess packages for WP 6.0 Beta 4

### PR DESCRIPTION
**Pull request by gziolo**
Trac ticket: https://core.trac.wordpress.org/ticket/55567

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: tests/phpunit/tests/blocks/renderCommentTemplate.php [hunk 1]
```diff
@@ -194,25 +194,27 @@ function test_rendering_comment_template() {
...
 
 	/**
-	 * Test rendering 3 nested comments:
+	 * Test rendering nested comments:
 	 *
 	 * └─ comment 1
 	 *    └─ comment 2
 	 *       └─ comment 3
+	 *       └─ comment 4
+	 *    └─ comment 5
```
Comment (gziolo) [inline tests/phpunit/tests/blocks/renderCommentTemplate.php:210]:
```suggestion
```

File: tests/phpunit/tests/blocks/renderCommentTemplate.php [hunk 2]
```diff
@@ -194,25 +194,26 @@ function test_rendering_comment_template() {
...
 	}
 
 	/**
-	 * Test rendering 3 nested comments:
+	 * Test rendering nested comments:
 	 *
 	 * └─ comment 1
 	 *    └─ comment 2
 	 *       └─ comment 3
+	 *       └─ comment 4
```
Comment (gziolo) [inline tests/phpunit/tests/blocks/renderCommentTemplate.php:209]:
```suggestion
	 *    └─ comment 4
```

**Review by adamziel — APPROVED** [review]

**Comment by kebbet** [comment]
Is there a fix planned for this issue? https://github.com/WordPress/gutenberg/issues/40675

**Comment by gziolo** [comment]
> Is there a fix planned for this issue? [WordPress/gutenberg#40675](https://github.com/WordPress/gutenberg/issues/40675)

It's listed on the board with fixes for 6.0 but there is no patch. It won't be included today.

**Comment by gziolo** [comment]
Committed with https://core.trac.wordpress.org/changeset/53329.

File: tests/phpunit/tests/blocks/renderCommentTemplate.php (hunk 1 above)
Comment (ockham) [reply tests/phpunit/tests/blocks/renderCommentTemplate.php:210]: Ah, looks like this didn't make it into `trunk`: https://github.com/WordPress/wordpress-develop/blob/a913b7312db9ea68bd86e9658ec69a97473bfb30/tests/phpunit/tests/blocks/renderCommentTemplate.php#L206-L210

Here's what the structure should look like: https://github.com/WordPress/gutenberg/blob/d0b600f44406d48fa8950fb3d519630aaf6bfea6/phpunit/class-block-library-comment-template-test.php#L149-L152

**Pull request closed without merging**